docs/UserGuide.md

```diff
@@ -6,7 +6,14 @@
 
 # SocialBook User Guide
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a  Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+SocialBook is a **desktop app specifically designed to ease the administrative process for social workers.**
+
+By providing a **comprehensive way to store the personal details** of those they are helping, as well as **functionalities 
+to make the information managing process efficient**, SocialBook streamlines the process for social workers. 
+
+SocialBook is **optimized for use via a  Line Interface** (CLI) while still having the benefits of a 
```

Comment:
Seems like you may have missed out "Command" when specifying Command Line Interface?